src/api/app/views/webui/user/edit.html.haml

```diff
@@ -8,15 +8,16 @@
   = "Editing User Data for User #{@displayed_user.login}"
 
 = form_tag(action: "save") do
-  %p
-    = hidden_field_tag("user", @displayed_user.login)
-    = label_tag(:realname, "Name:")
-    %br
-    = text_field_tag(:realname, @displayed_user.realname)
-  %p
-    = label_tag(:email, "e-Mail:")
-    %br
-    = text_field_tag(:email, @displayed_user.email, required: true, email: true)
+  - unless CONFIG[:ldap_mode]
```

Comment:
Nope it's reachable since you might want to adjust admin rights. The page keeps the same except the `realname` and `email` part.